docs/userguide.rst

```diff
@@ -315,6 +315,33 @@ and push the created image to the DockerHub image registry:
 
     $ docker push johndoe/myenv
 
+**Supporting non-root users**
```

Comment:
... or "Supporting arbitrary user IDs" as in the OpenShift docs, may appear psychologically more positive than "non-root" due to "non" :wink: